src/main/java/com/box/sdk/BoxAPIConnection.java

```diff
@@ -299,31 +311,37 @@ public void setTokenURL(String tokenURL) {
      * @return The url used for token revocation.
      */
     public String getRevokeURL() {
-        return this.revokeURL;
+        if (this.revokeURL != null) {
+            return this.revokeURL;
+        } else {
+            return this.baseURL + REVOKE_URL_SUFFIX;
+        }
     }
 
     /**
      * Set the URL used for token revocation.
      *
      * @param url The url to use.
+     * @deprecated Use {@link BoxAPIConnection#setBaseURL(String)}
      */
     public void setRevokeURL(String url) {
         this.revokeURL = url;
     }
 
     /**
      * Gets the base URL that's used when sending requests to the Box API. The default value is
```

Comment:
Maybe we should mention that it returns API_VERSION as well?
Just to inform that after invoke `setBaseURL` `getBaseURL` will return a different value.